CIPs/CIP-85/CIP-85.md

```diff
@@ -69,16 +73,16 @@ The `AppendCollection` schema must be an `object` with a `$ceramic` field from [
 
 #### CollectionSlice schema
 
-The `CollectionSlice` schema must be an `object` with a `$ceramic` field from [CIP-88](../CIP-88/CIP-88.md) having the type `collectionSlice` and pointing to the slice's `schema`, along with the following properties:
+The `CollectionSlice` schema must be an `object` with a `$comment` field from [CIP-88](../CIP-88/CIP-88.md) having the value `ceramic:collectionSlice`, along with the following properties:
 
-- `collection`: the DocID of the collection the slice is part of
+- `collection`: the StreamID of the collection the slice is part of
 - `sliceIndex`: index of the slice in the collection, between `0` and the collection's `slicesCount` minus `1`
 - `contents`: array with a `maxItems` value matching the `AppendCollection` `sliceMaxItems` property and defining the items schemas, that must include `{ type: 'null' }` in order to support removals
 
 ```js
 {
   $schema: 'http://json-schema.org/draft-07/schema#',
-  $ceramic: { type: 'collectionSlice' },
+  $comment: 'ceramic:collectionSlice',
```

Comment:
Why is this property needed here? I don't see any rationale for it. The fact that the schema is referenced from the append collection should be enough context for us to know that it's a collection slice.